config/README.md

```diff
@@ -0,0 +1,31 @@
+# Kustomizations for Baremetal Operator
+
+This folder contains kustomizations for the Baremetal Operator. They have
+traditionally been used through the [deploy.sh](../tools/deploy.sh) script,
+which takes care of generating the necessary config for basic-auth and TLS.
+However, a more GitOps friendly way would be to create your own static overlay.
+Check the `overlays/e2e` for an example that is used in the e2e tests.
+
+**NOTE** that you will need to supply the necessary secrets and config! This can
+be done in many ways, e.g. through the
+[external secrets operator](https://external-secrets.io/latest/) or directly in
+your overlay.
+
+- **base** - This is the kustomize base that we start from.
+- **components** - In here you will find re-usable kustomize components for TLS
+  and basic-auth.
+   - **basic-auth** - Enable basic authentication. Note that the basic-auth
+      component is missing the actual credentials. This is on purpose, to make
+      sure that the user is setting the password instead of using sample
+      credentials. The required secrets are:
+      - `ironic-credentials`
+      - `ironic-inspector-credentials`
+   - **tls** - Enable TLS. A CA certificate is needed here to verify the
+      connection to Ironic. If you deploy Ironic together with Ironic in a
```

Comment:
Do you mean 
```suggestion
      connection to Ironic. If you deploy Ironic-inspector together with Ironic in a
```

Reply:
Thanks good find. I mean Ironic together with BMO. Will fix!